WireAnalytics/Sources/WireAnalytics/Countly+AnalyticsService.swift

```diff
@@ -39,8 +39,12 @@ extension Countly: AnalyticsService {
         }
     }
 
-    public func trackEvent(_ event: AnalyticEvent) {
-        recordEvent(event.rawValue)
+    public func trackEvent(_ event: AnalyticsEvent) {
+        let segmentationDict = Dictionary(
+            uniqueKeysWithValues: event.segmentation.map { ($0.key, $0.value) }
+        )
```

Comment:
Isn't there some default segmentation that you need to send with every event, such as the app name and build info?